templates/admin/usergroup/list.html.twig

```diff
@@ -0,0 +1,95 @@
+        {% endif %}
+    </script>
+</head>
+<body class="body_top">
+<div class="container">
+    <div class="row">
+        <div class="col-xs-12">
+            <div class="page-title">
+                <h1>{{ 'User / Group'|translate }}</h1>
+            </div>
+        </div>
+    </div>
+    <div class="row">
+        <div class="col-xs-12">
+            <div class="btn-group">
+                <a href="usergroup_admin_add.php" data-toggle="modal" data-target="#modal-add-facility" class="btn btn-default btn-add">{{ 'Add User'|translate }}</a>
+                <a href="facility_user.php" class="btn btn-default btn-view">{{ 'View Facility Specific User Information'|translate }}</a>
+            </div>
+            <form name='userlist' method='post' class="form-inline pull-right" action='usergroup_admin.php' onsubmit='return top.restoreSession()'>
+                <div class="checkbox">
+                    <label for="form_inactive">
+                        <input type='checkbox' class="form-control" id="form_inactive" name='form_inactive' value='1' onclick='submit()' {% if form_inactive %}checked{% endif %}>
+                        {{ 'Include inactive users'|translate }}
+                    </label>
+                </div>
+            </form>
+        </div>
+    </div>
+    <div class="row">
+        <div class="col-xs-12">
+            {% if activeMsg == true %}
+            <div class="alert alert-danger">
+                {{ 'Emergency Login ACL is chosen. The user is still in active state, please de-activate the user and activate the same when required during emergency situations. Visit Administration->Users for activation or de-activation'|translate }}.
+            </div>
+            {% endif %}
+            {% if showMessage == true %}
+            <div class="alert alert-danger">{{ "The following Emergency Login User is activated:"|translate }} {{ fname }}</div>
+            <div class="alert alert-danger">{{ "Emergency Login activation email will be circulated only if following settings in the interface/globals.php file are configured"|translate }}: {{ global.emergency_login_email }}, {{ global.emergency_login_email_id }}</div>
+            {% endif %}
+
+            <table class="table table-striped">
+            <thead>
+                <tr>
+                    <th>{{ 'Username'|translate }}</th>
+                    <th>{{ 'Real Name'|translate }}</th>
+                    <th>{{ 'Additional Info'|translate }}</th>
+                    <th>{{ 'Authorized'|translate }}</th>
+                </tr>
+            <tbody>
+                {% for user in users %}
+                <tr>
+                    <td><a href="user_admin.php?id={{ user.id|e('url') }}" class="iframe_medium" onclick="top.restoreSession()">{{ user.username }}</a></td>
+                    <td>{{ user.fullName }}</td>
+                    <td>{{ user.info|translate }}</td>
+                    <td>{{ user.authorized|translate }}</td>
```

Comment:
Note there are definitely times where need to translate variables, but goal it to prevent this, if possible.
For example, if you end up using a service call to collect the user information, then seems like won't have a choice but to translate here (unless looped through the results in the model and processed stuff).